docs/contribute/01_environment.qmd

```diff
@@ -246,6 +246,47 @@ for manager, params in managers.items():
     * `Rebuild Container` to copy files from the local Git repo and have the build
     run `uv sync`.
 
+## uv
+
+::: {.callout-warning}
+## `uv` will not handle installation of system dependencies
+
+`uv` will not install system dependencies needed for some packages such as `psycopg2` and `kerberos`.
+
+For a better development experience see the `conda/mamba` or `nix` setup instructions.
+:::
+
+1. [Install `uv`](https://docs.astral.sh/uv/getting-started/installation/)
+
+1. [Install `gh`](https://cli.github.com/manual/installation)
```

Comment:
is gh only needed for the `gh repo fork --clone` call below? Or do the other justfile recipes require it? If it is only for this first step, I could see it being nice to removing this as a dependency: only instructing people to install uv, and then providing the raw `git` commands (would also need to instruct them to fork the repo, probably in the github web ui).

Reply:
Why would we want to replace a one liner with clicking on a web page?

Reply:
Sorry, I didn't mean to sound like a jerk there. After rereading what I wrote I realized it probably didn't come off exactly as I meant.

The original reason I chose gh is because it's the tool that lets you stay in one mode while setting everything up, whereas having people click to fork requires context switching. I don't see what we gain here by removing the gh dependency.

Reply:
Yeah no worrie!  Sounds good to me, just wanted to throw it out there